test/test_ssl.py

```diff
@@ -63,6 +64,21 @@ def test_create_urllib3_no_context(self) -> None:
             with pytest.raises(TypeError):
                 ssl_.create_urllib3_context()
 
+    def test_create_urllib3_context_default_verify_flags(self):
+        context = ssl_.create_urllib3_context()
+        if sys.version_info >= (3, 13):
+            assert context.verify_flags & ssl.VERIFY_X509_PARTIAL_CHAIN
+            assert context.verify_flags & ssl.VERIFY_X509_STRICT
+        else:
+            assert not (context.verify_flags & ssl.VERIFY_X509_PARTIAL_CHAIN)
+            assert not (context.verify_flags & ssl.VERIFY_X509_STRICT)
```

Comment:
Since we are setting the same flags as Python does, please consider taking an approach similar to `test_default_ssl_context_ssl_min_max_versions` where `verify_flags` value of a context created by Python is compared to one of the urllib3's context. This way we can detect future mismatches earlier.

https://github.com/urllib3/urllib3/blob/3e8f2db735dcaced6a3b777aa1966f40c018af7c/test/with_dummyserver/test_https.py#L1109-L1124

Reply:
I took this simple approach which I think will fail for PYOPENSSL? If so, I'm not sure how to fix that.
```
    def test_default_ssl_context_verify_flags(self) -> None:
        ctx = urllib3.util.ssl_.create_urllib3_context()
        ssl_ctx = ssl.create_default_context()
        assert ctx.verify_flags == ssl_ctx.verify_flags
```

Reply:
It works for PYOPENSSL when using `ssl.VERIFY_X509_TRUSTED_FIRST` as a default for `verify_flags`.